tools/CoqMakefile.in

```diff
@@ -762,11 +762,23 @@ $(GHTMLFILES): %.g.html: %.v %.glob
 # Dependency files ############################################################
 
 ifndef MAKECMDGOALS
-  -include $(ALLDFILES)
+	flagIncludeALLDFILES=1
 else
   ifneq ($(filter-out archclean clean cleanall printenv make-pretty-timed make-pretty-timed-before make-pretty-timed-after print-pretty-timed print-pretty-timed-diff print-pretty-single-time-diff,$(MAKECMDGOALS)),)
-   -include $(ALLDFILES)
- endif
+    flagIncludeALLDFILES=1
+  endif
+endif
+
+ifdef flagIncludeALLDFILES
+  VFILESNOTFOUND=$(filter-out $(wildcard $(VFILES)),$(VFILES))
+  ifneq ($(VFILESNOTFOUND),)
+    ifndef IGNOREMISSINGVFILE
+      $(warning W: Can't find $(VFILESNOTFOUND). Pass IGNOREMISSINGVFILE=1 to disable this check)
+      include $(ALLDFILES)
```

Comment:
This if should have a nonempty else clause dictating what happens when vfiles are not found and missing files are not ignored.  Alternatively you can drop this inner if statement and just do `include`

Reply:
I was under the impression that some exotic miss-configured makefiles might break, so I intend to `-include` in the `disabled check` case for backwards compatibility. Would you not do so?

Further, you said that I should `-include` if all VFILES are there to avoid "prevent a spurious message in the common case". I testet just using "include" in all cases and did not see any difference in output behaviour. What did you mean?

Reply:
I believe if the vdfile does not exist, you get an "error" message saying that it does not exist, and then it gets made, if you skip the initial dash.  This will in general only happen during a clean checkout.  I would suggest replacing `flagIncludeALLDFILES` with `SKIP_DINCLUDES` or similar, testing that this is not set to 1 (so that you can set it on the command line or in the common makefile), and then, if you want to avoid the message and still get errors, you can do something like $(shell $(MAKE) -f $(SELF) $(ALLDFILES) SKIP_DINCLUDES=1)`, I think.